src/mbgl/map/transform_state.cpp

```diff
@@ -340,6 +340,26 @@ double TransformState::getMaxZoom() const {
     return scaleZoom(max_scale);
 }
 
+void TransformState::setMinPitch(const double pitch_) {
+    if (pitch_ <= maxPitch) {
+        minPitch = util::clamp(pitch_, util::PITCH_MIN, maxPitch);
+    }
```

Comment:
should we report warning if pitch is less than util::PITCH_MIN?

Reply:
Added warning